xds/internal/balancer/clusterresolver/configbuilder_test.go

```diff
@@ -179,156 +178,10 @@ func TestBuildPriorityConfigJSON(t *testing.T) {
 	}
 }
 
+// TestBuildPriorityConfig tests the priority config generation. Each top level
+// balancer per priority should be an Outlier Detection balancer, with a Cluster
+// Impl Balancer as a child.
 func TestBuildPriorityConfig(t *testing.T) {
-	gotConfig, gotAddrs, _ := buildPriorityConfig([]priorityConfig{
```

Comment:
Should we retain this test and run it with the env var set to "false"?

Reply:
I'm fine with removing it given we would like to eliminate the env var entirely in a few weeks.